radixdlt-core/radixdlt/src/main/java/com/radixdlt/network/p2p/P2PConfig.java

```diff
@@ -145,6 +145,12 @@
    */
   Duration issuedProxyCertificateValidityDuration();
 
+  /** Specifies a list of peers that this node can connect to. If empty, all peers are allowed. */
+  ImmutableSet<NodeId> peerWhitelist();
```

Comment:
Can we call this AllowList/DenyList? See eg https://www.ncsc.gov.uk/blog-post/terminology-its-not-black-and-white  (and ditto throughout the review)

Reply:
Sure, I'll rename that. I would have never thought that someone might consider this a "casually pejorative wording" but I guess that's just signum temporis of the 21st century ;)